exporter/exporterparser/prometheus_test.go

```diff
@@ -33,22 +36,24 @@ func TestPrometheusExporter(t *testing.T) {
 	}{
 		{
 			config: `
-exporters:
```

Comment:
Changing this is a little worrying, does this mean that we shall no longer have sections?